src/firefox/index.js

```diff
@@ -14,26 +15,20 @@ import {getPrefs as defaultPrefGetter} from './preferences';
 import {getManifestId} from '../util/manifest';
 import {createLogger} from '../util/logger';
 import {default as defaultFirefoxConnector, REMOTE_PORT} from './remote';
-
-
-const log = createLogger(__filename);
-
-export const defaultFirefoxEnv = {
-  XPCOM_DEBUG_BREAK: 'stack',
-  NS_TRACE_MALLOC_DISABLE_STACKS: '1',
-};
-
-
-// Import flow types.
-
+//Import flow types
```

Comment:
nit: we generally put a space after the slashes for readable comments like `// Import flow types`. We don't have a lint rule for that because when you comment out some code temporarily you get yelled at which can be annoying.